lib/libc/mips/string/Makefile.inc

```diff
@@ -15,10 +15,13 @@ MDSRCS+= \
 .endif
 
 .if ${MACHINE_CPU:Mcheri}
+.if !${MACHINE_ABI:Mpurecap}
 CHERI_MDSRCS=	\
 		memcpy_c.c \
 		memmove_c.c \
 		memset_c.c
+.endif
+
 CHERI_MISRCS=	\
```

Comment:
I think this can also go and have `CHERI_MDSRCS` just become `MDSRCS+=`. It's only used to override the compiler used, and presumably has origins in the very dark early days of the CheriBSD port and LLVM support, but these days we have bsd.cpu.mk and the compat bits to do the right thing (and if they're not passing `-cheri` or equivalent then we really want to know about it).